test/jqLiteSpec.js

```diff
@@ -2013,9 +2013,16 @@ describe('jqLite', function() {
 
     it('should covert browser specific css properties', function() {
       expect(camelCase('-moz-foo-bar')).toBe('MozFooBar');
+      expect(camelCase('MozFooBar')).toBe('MozFooBar');
```

Comment:
Surely this is not correct. If there is no `-moz-` at the start then we should be lower casing the first character according to the requirements of this patch. In other words I expect the behaviour to be as described in these two expectations...

```js
expect(camelCase('-moz-foo-bar')).toBe('MozFooBar');
expect(camelCase('MozFooBar')).toBe('mozFooBar');
```